### PR TITLE
Add test covering SSL readiness check with self-signed cert

### PR DESCRIPTION
**Pull request by alenkacz**
Test covering the bugfix. Prior to that fix, the deployment never finishes.

JIRA issues: MARATHON-8430

### CODE REVIEW

File: tests/system/apps/__init__.py [hunk 1]
```diff
@@ -99,3 +99,7 @@ def ipv6_healthcheck(app_id=None, parent_group="/"):
         For more information about the nc options just run `docker run alpine nc --help`
     """
     return load_app('ipv6-healthcheck', app_id, parent_group)
+
+
+def readiness_ssl(app_id=None, parent_group="/"):
```
Comment (zen-dog) [inline tests/system/apps/__init__.py]: nit: I'm not quite happy with the name. How about `app-with-https-readiness-checks`?

Comment (alenkacz) [reply tests/system/apps/__init__.py]: done

**Comment by alenkacz** [comment]
@zen-dog I already had a green run but I'll done one more after the naming change :)

**Comment by alenkacz** [comment]
The failed tests are unrelated to this PR so I am going to merge

**Pull request merged**